src/components/common/layout.js

```diff
@@ -43,21 +53,19 @@ const pageStyles = css`
   }
 `;
 
-const Layout = ({ children }) => {
-  const data = useStaticQuery(graphql`
-    query SiteTitleQuery {
-      site {
-        siteMetadata {
-          title
-        }
-      }
-    }
-  `);
-
+const Layout = ({ children, includeResearchNavigation }) => {
   return (
     <>
-      <Global styles={globalStyles} />
-      <Header siteTitle={data.site.siteMetadata.title} />
+      <Global
+        styles={[
+          globalStyles,
+          includeResearchNavigation ? globalStylesDoubleHeader : "",
+        ]}
+      />
+      <Header
+        siteTitle="Employers Social Mobility Alliance"
```

Comment:
why are we ditching the `data.site.siteMetadata.title` ? I think it's makes sense to have this in the config file with other meta data like authors etc

Reply:
I changed this so the head would match the design. Since we're decided against using the CMS for content, it seems odd to pull titles, and the same for metadata.

Reply:
this isn't coming from the CMS though, it's coming from gatsby-config.js

Reply:
just seen that this is pulling from a local file, not the contentful (which I guess we're not using at all?) so I'll revert this and make the title change there